docs: Add documentation to run the RATS on a CF on KIND setup (#544)
cloudfoundry/routing-release: .github/CONTRIBUTING.md

- `/repo/scripts/docker/lint.bash`: This will run required linters.

> [!IMPORTANT]

> > If you are about to submit a PR, please make sure to run `./scripts/test-in-docker.bash` or `DB=postgres ./scripts/test-in-docker.bash` for MySQL and Postgres to ensure everything is tested in clean container. If you are developing, you can create a docker container first, then the only required script to run before testing your specific component is `build-binaries.bash`.

Reviewer: ```suggestion
> If you are about to submit a PR, please make sure to run `./scripts/test-in-docker.bash` (for MySQL) or `DB=postgres ./scripts/test-in-docker.bash` (for Postgres) to ensure everything is tested in a clean container. If you are developing, you can create a docker container first. In this case, the script `build-binaries.bash` must be executed before testing your specific component.
```

Author: Done
